Client/Classes/Recievers/ClientEventReciever.sqx

```diff
@@ -219,15 +219,15 @@ namespace Intrusion.Client
 		public static method OnUnitEnteredVehicle("_unit" as Object, "_role" as String, "_vehicle" as Object, "_turret" as Array)
 		{
 			private ["_intrusionVehicle" as IntrusionVehicle];
-
+			
 			_intrusionVehicle = _vehicle getVariable ["IntrusionVehicle", classNull];
```

Comment:
This get function was added by me to show you how to get the intrusion vehicle variable out of a vehicle. But this code should be in the ClientSquadHandler.SetUniform instead. Reason: keep logic in event reciever minimal, and have handlers do the logic. Unit, Group and Side checks are ok. The third parameter to SetUniform can then be just the vheicle, and not an intrusion vehicle.